controllers/drplacementcontrol_controller.go

```diff
@@ -479,12 +478,8 @@ func (r *DRPlacementControlReconciler) reconcileDRPCInstance(ctx context.Context
 		return ctrl.Result{}, fmt.Errorf("failed to get DRPolicy %w", err)
 	}
 
-	// Currently validation of schedule in DRPolicy is done here. When
-	// there is a reconciler for DRPolicy, then probably this validation
-	// has to be done there and can be removed from here.
-	err = r.validateSchedule(drPolicy)
-	if err != nil {
-		return ctrl.Result{}, err
+	if err := rmnutil.DrpolicyValidated(drPolicy); err != nil {
```

Comment:
Note for later: We need to update DRPC status with some information from this point for user consumption of errors.